src/performance/server.js

```diff
@@ -61,18 +60,12 @@ app.get('/performance/api/v1/environment', (req, res) => {
 */
 app.post('/api/v1/runload', async function (req, res) {
     try {
-        if (loadProcess) {
```

Comment:
We still need this check to prevent multiple load processes running against one project.
It should check `loadProcess[req.body.projectID]` (after making sure projectID is set!)